### ISSUE TITLE
"code" property in coverage.json

### ISSUE THREAD

**Issue by shiqp** [SitePen/remap-istanbul#79]
As we all know, remap-istanbul has supported "code" property in coverage.json:
`var jsText = fileCoverage.code || readFile(filePath);`

However, in my coverage.json, the content of "code" property missed the information about `//# sourceMappingURL=xxx.js.map`. As a result, when code `var match = sourceMapRegEx.exec(jsText)` runs, the `match` always `false` and I will get the error like `Could not find source map for: xxxxxxxxx`. 
Actually, if we do `var jsText = readFile(filePath)` and `var match = sourceMapRegEx.exec(jsText)`, it works well.

So I think the logic about `match` should like:

> if (!match) {
>     jsText = readFile(filePath);
>     match = sourceMapRegEx.exec(jsText);
>     if (!match) {
>           /* We couldn't find a source map, so will copy coverage after warning. */
>           warn(new Error('Could not find source map for: "' + filePath + '"'));
>           srcCoverage[filePath] = fileCoverage;
>           return;
>     }
> }

One more question: who knows why the content of 'code' property in coverage.json missed the inforamtion about `//# sourceMappingURL=xxx.js.map`?

**Comment by dylans** [comment]
@shiqp we started to address this issue by landing c6a9d48 (from #73). Does this solve the issue for you, or do you believe that your PR solves this issue differently or better? At a minimum we plan to land your test cases even if your other code changes are not needed after #73.

**Comment by shiqp** [comment]
@dylans I am sorry that #73 does not solve my issue. And my PR solves another issue about the wrong path in coverage.json. Thanks!

**Comment by jdonaghue** [comment]
Thanks @shiqp! I will go ahead and land your changes now. Thanks!

**Comment by jdonaghue** [comment]
closed by https://github.com/SitePen/remap-istanbul/pull/82
